wazo_confd/plugins/extension/middleware.py

```diff
@@ -28,3 +28,23 @@ def get(self, extension_id, tenant_uuids):
     def get_by(self, tenant_uuids, **criteria):
         model = self._service.get_by(tenant_uuids=tenant_uuids, **criteria)
         return self._schema.dump(model)
+
+    def parse_and_update(self, model, body, **kwargs):
```

Comment:
parse_and_update and find_updated_fields will end up in every middleware where we do an update. Move these functions to a base class or to a helper function

Reply:
I created a base class for the middlewares